sdk/storage/storage-blob/src/Lease.ts

```diff
@@ -0,0 +1,173 @@
+   *
+   * @static
+   * @param {(ContainerClient | BlobClient)} client
+   * @param {number} breakPeriod Break period
+   * @param {LeaseOperationOptions} [options={}] Optional options to configure lease management operations.
+   * @memberof LeaseClient
+   */
+  public static async breakAll(
+    client: ContainerClient | BlobClient,
+    breakPeriod: number,
+    options: LeaseOperationOptions = {}) {
+    const aborter = options.abortSignal || Aborter.none;
+    const operationOptions = {
+      abortSignal: aborter,
+      breakPeriod,
+      modifiedAccessConditions: options.modifiedAccessConditions
+    };
+    const clientContext = new StorageClientContext(client.url, client.pipeline.toServiceClientOptions());
+    if (client instanceof ContainerClient) {
+      const container = new Container(clientContext);
+      await container.breakLease(operationOptions);
+    } else {
+      const blob = new Blob(clientContext);
+      await blob.breakLease(operationOptions);
+    }
+  }
+
+  /**
+   * Creates an instance of LeaseClient.
+   * @param {string} _leaseId Initial proposed lease id.
+   * @param {(Container | Blob)} containerOrBlob The object that is used to make the lease operation requests.
+   * @memberof LeaseClient
+   */
+  constructor(private _leaseId: string, private readonly containerOrBlob: Container | Blob) {
+  }
+
+  /**
+   * Establishes and manages a lock on a container for delete operations, or on a blob
+   * for write and delete operations.
+   * The lock duration can be 15 to 60 seconds, or can be infinite.
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/lease-container
+   * and
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/lease-blob
+   *
+   * @param {number} duration Must be between 15 to 60 seconds, or infinite (-1)
+   * @param {LeaseOperationOptions} [options={}] Optional option to configure lease management operations.
+   * @memberof LeaseClient
+   */
+  public async acquire(duration: number, options: LeaseOperationOptions = {}) {
+    const aborter = options.abortSignal || Aborter.none;
+    const proposedLeaseId = this._leaseId;
+    const response = await this.containerOrBlob.acquireLease({
+      abortSignal: aborter,
+      duration,
+      modifiedAccessConditions: options.modifiedAccessConditions,
+      proposedLeaseId
+    });
+
+    if (!response.errorCode && response.leaseId) {
+      this._leaseId = response.leaseId!;
+    } else {
+      throw new Error("Error changing lease Id to " + proposedLeaseId + ", error code: '" + response.errorCode + "'.");
+    }
+  }
+
+  /**
+   * To change the ID of the lease.
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/lease-container
+   * and
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/lease-blob
+   *
+   * @param {string} proposedLeaseId
+   * @param {LeaseOperationOptions} [options={}] Optional option to configure lease management operations.
```

Comment:
"Optional option" is funny. Maybe just say "options".

Reply:
Maybe 'options to configure lease management operations [Optional]'